app/Models/Share.php

```diff
@@ -51,7 +51,7 @@ public static function load($filename) {
 
 	/**
 	 * Return the list of sharing options.
-	 * @return an array of FreshRSS_Share objects.
+	 * @return array[FreshRSS_Share]
```

Comment:
I thought the syntax was ```FreshRSS_Share[]```

Reply:
I think there is more than one syntax, and I am not sure which is most common / best (although I think that `array`  is more usual than `[]`).
Feel free to change.
https://stackoverflow.com/questions/778564/phpdoc-type-hinting-for-array-of-objects/23072048#23072048
https://stackoverflow.com/questions/2713710/comment-associative-array-in-php-documentor/2725073